File: R/validate_user_input.R
library(stringr)
{
  words <- readline(prompt="Give me a word, any word: ")
  while(!grepl("^[A-Za-z]+$", words, perl=T)){
    words <- readline(prompt="I said a word! Try again. Give me a word, any word: ")
  } 
  print(paste("Hey! ",str_to_title(words)," is a cool word!", sep=""))
}

File: Python/validate_user_input.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Thu Jul  9 09:42:00 2020

@author: tylermanderfield
"""

a = input("Give me a word, any word: ")
while a.isalpha() == False:
    a = input("I said a word! Try again. Give me a word, any word: ")
print("Hey! " + a.capitalize() + " is a cool word!")